lib/abstract-browser.js

```diff
@@ -11,6 +11,11 @@ function AbstractBrowser () {
   EventEmitter.call(this)
 }
 
+AbstractBrowser.prototype.start = function () {
+  this.stats = { passed: 0, failed: 0 }
+  this._start()
```

Comment:
For the sake of unit testing, l think we should define a default noop `AbstractBrowser.prototype._start`.

Then we can write an abstract test suite, to run against the implementations (`SauceBrowser` etc) as well as `AbstractBrowser` itself.

Reply:
I'll update and squash after that.